contribute/backend/errors.md

```diff
@@ -1,26 +1,25 @@
 # Errors
 
-Grafana introduced its own error type [github.com/grafana/grafana/pkg/util/errutil.Error](../../pkg/util/errutil/errors.go)
-in June 2022. It's built on top of the Go `error` interface extended to
-contain all the information necessary by Grafana to handle errors in an
-informative and safe way.
+Grafana includes its own error type [github.com/grafana/grafana/pkg/util/errutil.Error](../../pkg/util/errutil/errors.go). 
+Introduced in June 2022, the type is built on top of the Go `error` interface.
+It extends the interface to contain all the information necessary by Grafana to handle errors in an informative and safe way.
 
-Previously, Grafana has passed around regular Go errors and have had to
+Previously, Grafana passed around regular Go errors and therefore had to
 rely on bespoke solutions in API handlers to communicate informative
 messages to the end-user. With the new `errutil.Error`, the API handlers
-can be slimmed as information about public messaging, structured data
-related to the error, localization metadata, log level, HTTP status
-code, and so forth are carried by the error.
+can be streamlined. The error carries information about public messaging, 
+structured data related to the error, localization metadata, log level, 
+HTTP status code, and so forth.
 
 ## Basic use
```

Comment:
use of double header here - each header should have text under it

Reply:
fixed by deleting header and moving subsidiary headers up